src/main/promotion/dto/promotion-create.dto.ts

```diff
@@ -0,0 +1,51 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import {
+    IsMilitaryTime,
+    IsNotEmpty,
+    max,
+    Max,
+    Min,
+    Validate,
+    ValidateNested,
+    validateOrReject,
+} from 'class-validator';
+import { OneToMany } from 'typeorm';
+
+export class PromotionCreateDTO {
+
+    @IsNotEmpty()
+    @Validate(Max(15))
```

Comment:
I remember Max(15) not @Validate(Max(15)) read documents control + f search Max() : https://github.com/typestack/class-validator#validation-errors